pkg/types/types.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+// TODO: User can put values like --port "mariadb=3306:3306"
```

Comment:
Here we add things we want to do like get information that we ask in interactive mode?